### PR TITLE
Add Detroit Arcology DAO to request re-indexing

### PR DESCRIPTION
**Pull request by cryptogoth**
Please add Detroit Arcology DAO (deployed with the addresses in this JSON) to subgraph

### CODE REVIEW

**Comment by orenyodfat** [comment]
could you please add that with the dao.json format .
see reference : https://github.com/daostack/subgraph/blob/master/daos/mainnet/general-store-dao.json

**Comment by cryptogoth** [comment]
Thanks @orenyodfat i made the requested changes. Let me know if you need anything else.

**Comment by cryptogoth** [comment]
In appreciation of your help in this last mile of enabling Detroit Arcology DAO, here's a preview of our physical DAO in Detroit
https://bit.ly/334Ny7a

File: daos/mainnet/detroit-arcology.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+        "name": "DetroitDAO",
+        "Avatar": "0xf931FF012CBff471CccB90A784E3E88B512c890A",
+        "DAOToken": "0xe6448C6502F41ee2949e0d7Fa7eb3c8846f909F9",
+        "Reputation": "0x68cCB9a589012dB8C697c3e9e1908C2aDEEC4BF4",
+        "Controller": "0x9868f32D9A2C3533b56D718deec68cDDc2E55A51",
```
Comment (orenyodfat) [inline daos/mainnet/detroit-arcology.json]: looks like this is a uController (not Controller) .
https://etherscan.io/address/0x9868f32D9A2C3533b56D718deec68cDDc2E55A51#code
right ?
in that case you do not need to mention that on the Dao.json.

**Comment by cryptogoth** [comment]
Yes, I used Dao creator and can provide the migration Params if that's helpful. What's  the difference between uController and controller?

Thanks for weekend response

**Comment by orenyodfat** [comment]
> Yes, I used Dao creator and can provide the migration Params if that's helpful. What's  the difference between uController and controller? Thanks for weekend response
> […](#)

see https://github.com/daostack/arc/blob/master/docs/contracts/controller/README.md#interface--implementations

**Comment by cryptogoth** [comment]
Thanks, yes it looks like DAOcreator chose a UController, which is fine for our current purposes. I'll remove that line from the JSON file.

**Comment by cryptogoth** [comment]
Thank you @orenyodfat . What's the next step to move this forward?

**Comment by orenyodfat** [comment]
> Thank you @orenyodfat . What's the next step to move this forward?

1. merge
2. index that 
3. register that so it will be shown on alchemy .

**Comment by orenyodfat** [comment]
@cryptogoth the dao is indexed and registered . see 
it will be shown in alchemy when it will shift to https://thegraph.com/explorer/subgraph/daostack/v29_0  .
hopefully soon

**Comment by orenyodfat** [comment]
@cryptogoth https://alchemy.daostack.io/dao/0xf931ff012cbff471cccb90a784e3e88b512c890a

**Pull request merged**